### PR TITLE
Added circleci terraform script

### PR DESCRIPTION
**Pull request by JKashyap96**

### CODE REVIEW

File: software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt [hunk 1]
```diff
@@ -26,6 +26,16 @@ target_branch, repository_name, commit_id
 | toLong(datetime) as datetime_epoch
 | "deploy" as event_type
 "
+      },
+       "CircleCI":{
+        "Scope" : "_sourcecategory=circleci/job-collector",
+        "Parse" : "json "workflows.job_name" as job_type
+|where toLowerCase(job_type) matches ("deploy*")
```
Comment (ankitgoelcmu) [inline software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt:33]: it should be `*deploy*`

File: software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt [hunk 2]
```diff
@@ -106,6 +116,16 @@ commit_id, reviewers, closeddate nodrop
 | where status in ("SUCCESSFUL", "FAILED")
 | "build" as event_type
 "
+      },
+      "CircleCI":{
+        "Scope" : "_sourcecategory=circleci/job-collector",
+        "Parse" : "| json "workflows.job_name" as job_type
+|where toLowerCase(job_type) matches ("build*")
```
Comment (ankitgoelcmu) [inline software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt:123]: same, it should be `*build*`

File: software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt (hunk 1 above)
Comment (JKashyap96) [reply software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt]: have updated the code

File: software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt (hunk 2 above)
Comment (JKashyap96) [reply software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt]: have updated the code

**Pull request merged**